lib/libcompat.h

```diff
@@ -89,6 +89,16 @@
 #include <process.h>            /* getpid */
 #endif /* _MSC_VER */
 
+#ifdef _WIN32
+#define CK_FMT_ZU "%Iu"
```

Comment:
Maybe leave a comment here saying why this is necessary, and maybe give a URL
as a reference.